WTWSMS/localisation/WtWSMS_titles.csv

```diff
@@ -17,7 +17,8 @@ PRAEFECTUS_URBI;Praefectus Urbi;Praefectus Urbi;;;;;;;;;;x
 EMPEROR;Emperor;Empereur;;;;;;;;;;x
 EMPRESS;Empress;Imp�ratrice;;;;;;;;;;x
 COMES_ORIENTIS;Comes Orientis;Comes Orientis;;;;;;;;;;x
-COMES_ORIENTIS_EMPEROR;Imperator Orientis;;;;;;;;;;x
+COMES_ORIENTIS_EMPEROR;Gloriosissimus Orientis;;;;;;;;;;x
+COMES_ORIENTIS_EMPEROR_FOA;Gloriosa Ductor;Gloriosa Ductor;;;;;;;;;;x
```

Comment:
I found someone who was literally fluent in Latin and could speak it conversationally. That was the title she suggessted as the title for am Empire of Oriens.